template/.travis.yml

```diff
@@ -0,0 +1,81 @@
+sudo: false
+
+language: php
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - $HOME/.local
+
+env:
+  global:
+    - COMPOSER_ARGS="--no-interaction"
+    - COVERAGE_DEPS="satooshi/php-coveralls"
+    - LEGACY_DEPS="phpunit/phpunit"
```

Comment:
Since this is for _new_ packages, I think we can ignore the `LEGACY_DEPS` situation entirely.